### ISSUE TITLE
Q: Fast components

### ISSUE THREAD

**Issue by leolorenzoluis** [JordanMarr/fable-lit-fullstack-template#10]
How's your experience with FAST components? It feels like it's buggy. Updating the value for a simple text-field does not update the next time but the correct value is set at runtime.

Edit: Looks like I just don't know what's the correct property to set. Changing `value` to `.value` does the trick.

**Comment by leolorenzoluis** [comment]
@JordanMarr Would like to know your reasoning why you have ShoeLace/FAST components though? Are there components in FAST that is just not enough or extensible as ShoeLace is?

**Comment by JordanMarr** [comment]
You don't have to use them if you don't want to.
I just included them both as a starting point to demonstrate how to cherry pick components.

**Comment by leolorenzoluis** [comment]
Understood. I'll share my experience. It looks like the text input from FAST component has a slower debounce hence when I type fast the input shows correct value but when I pass use the other component it works fine, so just want to know if you experience any other weirdness with FAST components :)

**Comment by JordanMarr** [comment]
Ah I see.
Yes, I had a large form with many Fluent UI (which is build on FAST) inputs until I started having strange issues/bugs with them.
I think that might why I started looking around for a different library, which is when I switched to using Shoelace inputs.

The Shoelace team was also very helpful when I posted [a question](https://github.com/shoelace-style/shoelace/issues/839) on their issues forum.

I would probably be using only Shoelace except for the fact that there are a few unique components in the Fluent UI framework that are really nice that don't exist in Shoelace, like the [Slider](https://docs.microsoft.com/en-us/fluent-ui/web-components/components/slider?pivots=typescript), for example.
